sqlserver/assets/configuration/spec.yaml

```
@@ -287,6 +287,85 @@ files:
           value:
             type: number
             example: 10
+    - name: aws
+      description: |
+        Provide the following if you are instrumenting a managed AWS database. Requires `dbm:true`.
+        
+        Requires setting the field `hostname`, which should be equal to the Endpoint.Address of the instance
+        the agent is connecting to. For example:
+        aws:
+          - hostname: foo.bar.us-east-1.rds.amazonaws.com
+        
+        Note, if the `host` field is already equal to the instance's Endpoint.Address value, then setting 
+        this configuration is not necessary.
+        
+        For more information on instance endpoints, 
+        see the AWS docs https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_Endpoint.html
+      value:
+        type: array
+        items:
+          anyOf:
+            - type: string
+            - type: object
+          properties:
+            - name: hostname
+              type: string
+          example:
+            - hostname: foo.bar.us-east-1.rds.amazonaws.com
```

I'm a little confused by the array. Since this is for a single instance, wouldn't the structure be:

```suggestion
      options:
        - name: hostname
          description: ...
          value:
            type: string
            example: foo.bar.us-east-1.rds.amazonaws.com
```

---

updated 👍